### PR TITLE
NTUEEPLUS-266. Update & Delete for frontend column

### PR DESCRIPTION
**Pull request by weiber2002**
### What is this PR for?

To implement functions of update and delete for columns in column_summary
### What type of PR is it?
[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring]
improve
### Todos
* [x] - update
* [x] - delete

### What is the Jira issue?

https://ntueeplus.atlassian.net/browse/NTUEEPLUS-266

### CODE REVIEW

**Comment by weiber2002** [comment]
image

**Comment by weiber2002** [comment]
image

File: client/src/views/auth/column/ColumnForm.js [hunk 1]
```diff
@@ -29,23 +30,25 @@ import {
 } from '@coreui/react'
 import axios from 'axios'
 import CIcon from '@coreui/icons-react'
+import { useParams, useLocation } from 'react-router-dom'
```
Comment (vwvwMM) [inline client/src/views/auth/column/ColumnForm.js:33]: why do you need useLocation?

File: client/src/views/auth/column/ColumnForm.js [hunk 2]
```diff
@@ -29,23 +30,25 @@ import {
...
 // import  CDatePicker  from '@coreui/react-pro'
 
 // import packagename.classname;
 // import java.lang.Math;
```
Comment (vwvwMM) [inline client/src/views/auth/column/ColumnForm.js:37]: useless comment code can be deleted

File: client/src/views/auth/column/ColumnForm.js (hunk 1 above)
Comment (vwvwMM) [reply client/src/views/auth/column/ColumnForm.js:33]: and you duplicate import things from 'react-router-dom'

**Review by vwvwMM — APPROVED** [review]
LGTM

**Pull request merged**